.github/workflows/create_prerelease.yaml

```diff
@@ -41,7 +41,65 @@ jobs:
           docker manifest push $DOCKERHUB_ORGANISATION/lorabasicsstationmodule:$VERSION
           docker manifest push $DOCKERHUB_ORGANISATION/lorawannetworksrvmodule:$VERSION
     env:
-      DOCKERHUB_ORGANISATION: ${{ secrets.DOCKER_REPOSITORY }}  
+      DOCKERHUB_ORGANISATION: ${{ secrets.DOCKER_REPOSITORY }}
+      
+  CreatePRToUpdateDoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: docs/main
+      - name: Update doc URL with new version
+        run: sed -r -i "s/iotedge-lorawan-starterkit%2Fv[0-9\.]+/iotedge-lorawan-starterkit%2Fv${VERSION}/g" docs/quickstart.md
+      - name: Set up git user using the built-in token
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Create branch
+        run: |
+          BRANCH_NAME="docs/release-${VERSION}-${{ github.run_id }}"
+          DESCRIPTION="Update button URL with version $VERSION."
```

Comment:
Why do you need to put those in the github environment? Isn't that enough to have them as bash var?

Reply:
Because I reuse the var in the next step.